Firmware/RTK_Everywhere/bluetoothSelect.h

```diff
@@ -33,7 +33,7 @@ class BTClassicSerial : public virtual BTSerialInterface, public BluetoothSerial
   public:
     bool begin(String deviceName, bool isMaster, uint16_t rxQueueSize, uint16_t txQueueSize)
     {
-        return BluetoothSerial::begin(deviceName, isMaster, rxQueueSize, txQueueSize);
+        return BluetoothSerial::begin(deviceName, isMaster); //, rxQueueSize, txQueueSize); v3.0.0 has no QueueSize parameters
```

Comment:
v2 did not have QueueSize either. It was a custom add to the stock library to allow variable buffer sizes within the BluetoothSerial library. We have removed it in the past which lead to problems. I suspect we will run into those same problems but it's trivial to add it back at that time.

Reply:
OK - thanks - duly noted...